capa/rules/cache.py

```diff
@@ -159,3 +159,25 @@ def load_cached_ruleset(cache_dir: Path, rule_contents: List[bytes]) -> Optional
         return None
     else:
         return cache.ruleset
+
+
+def generate_rule_cache(rules_dir: Path, cache_dir: Path) -> bool:
+    if not rules_dir.is_dir():
+        logger.error(f"rules directory '{rules_dir}' does not exist")
+        return False
+
+    try:
+        cache_dir.mkdir(parents=True, exist_ok=True)
+        rules = capa.rules.get_rules([rules_dir], cache_dir)
+    except (IOError, capa.rules.InvalidRule, capa.rules.InvalidRuleSet) as e:
+        logger.error(f"{str(e)}")
```

Comment:
```suggestion
        logger.error(f"{e:s}")
```

I think this is correct and more succinct, but maybe your way is more readable.

Reply:
flake8 would complain for both of these :')
`capa/rules/cache.py:173:22: G004 Logging statement uses f-string`

Reply:
🤦 of course